docs/contributing/contributing-to-nextJS-branch.md

```diff
@@ -0,0 +1,164 @@
+- [Fork the Repo](https://github.com/EddieHubCommunity/LinkFree/fork). Then, create a branch for any issue that you are working on. Finally, commit your work.
+- Create a **[Pull Request](https://github.com/EddieHubCommunity/LinkFree/compare)** (_PR_), which will be promptly reviewed and given suggestions for improvements by the community.
+- Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes proposed in your PR.
+
+---
+
+## ⭐ HOW TO MAKE A PULL REQUEST:
+
+**1.** Start by making a Fork of the [**LinkFree**](https://github.com/EddieHubCommunity/LinkFree) repository. Click on the <a href="https://github.com/EddieHubCommunity/LinkFree/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a>Fork symbol at the top right corner.
+
+Dont forget to uncheck the 'Copy the main branch only' option when forking the repo.
+
+![Screenshot 2022-10-05 235606](https://user-images.githubusercontent.com/32553276/194235257-5b53c801-514f-48d9-ba22-31352c24373c.png)
+
+**2.** Clone your new fork of the repository in the terminal/CLI on your computer with the following command:
+
+```bash
+git clone https://github.com/<your-github-username>/LinkFree
+```
+
+**3.** Navigate to the newly created LinkFree project directory:
+
+```bash
+cd LinkFree
+```
+
+**4.** Set upstream command:
+
+```bash
+git remote add upstream https://github.com/EddieHubCommunity/LinkFree.git
+```
+
+**5.** Create a new branch:
+
+```bash
+git checkout -b YourBranchName
+```
+
+**6.** Sync your fork or your local repository with the origin repository:
+
+- In your forked repository, click on "Fetch upstream"
+- Click "Fetch and merge"
+
+### Alternatively, Git CLI way to Sync forked repository with origin repository:
+
+```bash
+git fetch upstream
+```
+
+```bash
+git merge upstream/nextjs
+```
+
+### [Github Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for Syncing
+
+**7.** Make your changes to the source code.
+
+**8.** Stage your changes:
+
+⚠️ **Make sure** not to commit `package.json` or `package-lock.json` file
+
+⚠️ **Make sure** not to run the commands `git add .` or `git add *`
+
+> Instead, stage your changes for each file/folder
+>
+> By using public path it means it will add all files and folders under that folder, it is better to be specific
+
+```bash
+git add public
+```
+
+_or_
+
+```bash
+git add "<files_you_have_changed>"
+```
+
+**9.** Commit your changes:
+
+```bash
+git commit -m "<your_commit_message>"
+```
+
+**10.** Push your local commits to the remote repository:
+
+```bash
+git push origin YourBranchName
+```
+
+**11.** Create a [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)!
```

Comment:
Maybe worth adding here to check that the base branch is Linkfree/next-js branch and not main